tikv-client/src/main/java/com/pingcap/tikv/Snapshot.java

```diff
@@ -69,14 +70,21 @@ public ByteString get(ByteString key) {
         .get(key, timestamp.getVersion());
   }
 
-  public List<KvPair> batchGet(List<byte[]> keys) {
+  public List<BytePairWrapper> batchGet(List<byte[]> keys) {
```

Comment:
why should we change this external interface? maybe other applications are using it.